pilz_control/test/config/joint_limits.yaml

```diff
@@ -0,0 +1,40 @@
+#
+# Copyright © 2020 Pilz GmbH & Co. KG
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# joint_limits.yaml allows the dynamics properties specified in the URDF to be overwritten or augmented as needed
+# Specific joint properties can be changed with the keys [max_position, min_position, max_velocity, max_acceleration]
+# Joint limits can be turned off with [has_velocity_limits, has_acceleration_limits]
+joint_limits:
+  shoulder_to_right_arm:
+    has_velocity_limits: true
+    max_velocity: 1.57
+    has_acceleration_limits: true
+    max_acceleration: 3.49
+  shoulder_to_left_arm:
+    has_velocity_limits: true
+    max_velocity: 1.57
+    has_acceleration_limits: true
+    max_acceleration: 3.49
+  joint_with_undefined_max_acc:
+    has_velocity_limits: true
+    max_velocity: 1.57
+    has_acceleration_limits: true
+  joint_with_has_acc_lim_false:
+    has_velocity_limits: true
+    max_velocity: 1.57
+    has_acceleration_limits: false
+  joint_with_undefined_has_acc_lim:
```

Comment:
Joint Name doesn't explicitely state that `max_acceleration` is also undefined.

Reply:
Added `max_acceleration` entry